doc/Overview/KeyEquality/concept.md

```diff
@@ -99,11 +101,11 @@ public partial record MyItem(Guid EntityId, string SourceId, string Value);
 
 ## How to disable generation?
 
-You can disable the generation on a given type by adding `[ImplicitKeyEquality(IsEnabled = false)]` on it.
+You can disable the generation on a given type by adding `[ImplicitKeysEquality(IsEnabled = false)]` on it.
 
 To disable the generation for the whole project you set that attibute directly on the assembly:
 ```csharp
-[assembly:ImplicitKeyEquality(IsEnabled = false)]
+[assembly:ImplicitKeysEquality(IsEnabled = false)]
```

Comment:
@dr1rrb The word "Equality" is not actually in the class name.

https://github.com/unoplatform/uno.extensions/blob/c439932faabf3f3287c5f307e9c01601034b0b6a/src/Uno.Extensions.Core/Equality/ImplicitKeysAttribute.cs#L20